src/Components/Classes/List/ClassList.jsx

```diff
@@ -1,77 +1,29 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom/cjs/react-router-dom.min';
 import styles from './list.module.css';
-import ResponseModal from '../../Shared/ResponseModal';
 import Button from '../../Shared/Button';
 import ConfirmModal from '../../Shared/ConfirmModal';
 import { Input } from '../../Shared/Inputs';
 
-function ClassList({ classes, getById, selectedClass, setRenderData }) {
+function ClassList({ classes }) {
```

Comment:
The classes prop here I think that you don't need it anymore, because you have it in the global state. So you can use useSelector here and gain the list from there